Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -12,7 +12,6 @@
 using MediaBrowser.Controller.Net.WebSocketMessages;
 using MediaBrowser.Controller.Net.WebSocketMessages.Outbound;
 using MediaBrowser.Model.Session;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 
 namespace Emby.Server.Implementations.HttpServer
```

Comment:
Are we planning switching to the file-scoped namespace everywhere?

Reply:
Maybe, but I wouldn't go and change them as that would mess with git blame

Reply:
@Bond-009, At least in jetbrains rider it doesn't affect git blame at all. But I am not sure about VS and git cli.